app/controllers/idv/please_call_controller.rb

```diff
@@ -22,5 +23,10 @@ def confirm_fraud_pending
         redirect_to account_url
       end
     end
+
+    def in_person_proofing?
+      IdentityConfig.store.in_person_proofing_enabled && 
+        current_user&.pending_profile&.in_person_enrollment.status == 'passed'
```

Comment:
We saw that they're necessarily equivalent here, but should we use `fraud_review_pending_profile` instead of `pending_profile` for consistency with below?